pyfar/constants/speed_of_sound.py

```diff
@@ -52,3 +52,113 @@ def speed_of_sound_simple(temperature):
     t_0 = pf.constants.absolute_zero_celsius
     c_ref = pf.constants.reference_speed_of_sound
     return c_ref*np.sqrt((temperature-t_0)/(t_ref-t_0))
+
+
+
+def speed_of_sound_cramer(
+        temperature, relative_humidity, atmospheric_pressure=None,
+        co2_ppm=314):
+    """
+    Calculate the speed of sound in air based on temperature, atmospheric
+    pressure, humidity and CO2 concentration.
+
+    This implements Cramers method described in [#]_.
+    The uncertainty in the speed of sound is similarly estimated to be
+    less than 300 parts per million.
+
+    Parameters
+    ----------
+    temperature : float, array_like
+        Temperature in degree Celsius from 0°C to 30°C.
+    relative_humidity : float, array_like
+        Relative humidity in the range of 0 to 1.
+    atmospheric_pressure : float, array_like, optional
+        Atmospheric pressure in pascal, by default
```

Comment:
Harmonize within constants module? It could be "Pa" as well.
```suggestion
        Atmospheric pressure in Pascal, by default
```